wallet/cardano-sl-wallet.cabal

```diff
@@ -181,6 +181,7 @@ library
   ghc-options:         -Wall
                        -fno-warn-orphans
                        -O2
+                       -fwarn-redundant-constraints
```

Comment:
Didn't know about that. I suspect that will be useful!

Reply:
Actually, I think that flag was part of `-Wall` in ghc 8.0.1 but got removed in 8.0.2

Reply:
i tend to use `-Weverything`, even more then all!

Reply:
`-Wno-really-everything`